scripts/dataCreation.js

```diff
@@ -10,15 +10,22 @@ exports.execute = () => {
     password: '$2b$10$FoKOu6OIHD20/6C8E86YjueiahPtZwSxLFRJJucC7o0wY3bkVmXHS',
     updatedAt: '2018-10-24T18:51:59.125Z',
     createdAt: '2018-10-24T18:51:59.125Z'
-  }).then(() => {
-    return User.create({
-      name: 'Federico',
-      surname: 'NotAdmin',
-      email: 'federico.notadmin@wolox.com.ar',
-      admin: false,
-      password: '$2b$10$FoKOu6OIHD20/6C8E86YjueiahPtZwSxLFRJJucC7o0wY3bkVmXHS',
-      updatedAt: '2018-10-24T18:51:59.125Z',
-      createdAt: '2018-10-24T18:51:59.125Z'
+  })
+    .then(() => {
+      return User.create({
+        name: 'Federico',
+        surname: 'NotAdmin',
+        email: 'federico.notadmin@wolox.com.ar',
+        admin: false,
+        password: '$2b$10$FoKOu6OIHD20/6C8E86YjueiahPtZwSxLFRJJucC7o0wY3bkVmXHS',
+        updatedAt: '2018-10-24T18:51:59.125Z',
+        createdAt: '2018-10-24T18:51:59.125Z'
+      });
+    })
+    .then(() => {
+      return Purchase.create({
+        userId: 1,
```

Comment:
you shouldn't use wired ids, you should receive the use as a params in the then and use the given user id.

Reply:
Fixed